internal/cmd/identities.go

```diff
@@ -289,14 +280,19 @@ func promptPasswordConfirm(oldPassword string) (string, error) {
 		{
 			Name:     "Confirm",
 			Prompt:   &survey.Password{Message: "Confirm Password:"},
-			Validate: checkConfirmPassword(&passwordConfirm.Password),
+			Validate: checkConfirmPassword(),
```

Comment:
```suggestion
```

Looks like this validation should be removed from the survey input completely if it happens after the input

Reply:
done